source/module_base/test/ylm_test.cpp

```diff
@@ -0,0 +1,33 @@
+#include "../ylm.h"
+#include "gtest/gtest.h"
+/************************************************
+ *  unit test of class ylm
+ ***********************************************/
+
+/**
+ * - Tested Functions:
+ *   - ZEROS
+ *     - set all elements of a double float array to zero
+ * */
+
+class ylmTest : public testing::Test
+{
+	
+};
+
+TEST_F(ylmTest,Zeros)
+{
+    double aaaa[100];
+    ModuleBase::Ylm::ZEROS(aaaa,100); 
+    EXPECT_EQ(aaaa[0],0.0);

```

Comment:
Please delete repeated EXPECTs.

Reply:
ok, I have already finished it. Please check it again.